.github/workflows/check-docs.yaml

```diff
@@ -1,5 +1,8 @@
 name: "Check documentation"
 on:
+  push:
+    branches:
+    - abailly-iohk/fix-link-checker-job
```

Comment:
I think you would want to remove this again.

Reply:
definitely, but it's actually not working :(
https://github.com/input-output-hk/hydra-poc/actions/runs/3081422680/jobs/4979948978